deploy.sh

```diff
@@ -40,20 +46,16 @@ echo -e "Generated docs\n"
 # Change directory to $DOCS_DIR
 cd $DOCS_DIR
 
-# Flatten content of $DOCS_DIR
-mv $PACKAGE_NAME/**/** ./
```

Comment:
Nope. It was needed to cleanup the directory mess caused by JSDoc when passing a package.json.